web3swift/src/Extensions/HexExtensions.swift

```diff
@@ -70,28 +70,38 @@ public extension Web3Extensions where Base == String {
         }
         return base
     }
-    
+
     var withHexPrefix: String {
         if !base.hasPrefix("0x") {
             return "0x" + base
         }
         return base
     }
-    
+
     var stringValue: String {
         if let byteArray = try? HexUtil.byteArray(fromHex: base.web3.noHexPrefix), let str = String(bytes: byteArray, encoding: .utf8) {
             return str
         }
-        
+
         return base
     }
-    
+
     var hexData: Data? {
         let noHexPrefix = self.noHexPrefix
         if let bytes = try? HexUtil.byteArray(fromHex: noHexPrefix) {
-            return Data( bytes)
+            return Data(bytes)
         }
-        
+
         return nil
     }
 }
+
+public extension String {
+    func stripHexPrefix() -> String {
+        if hasPrefix("0x") {
+            let indexStart = index(startIndex, offsetBy: 2)
+            return String(self[indexStart...])
+        }
+        return self
+    }
+}
```

Comment:
We already have an extension on string `noHexPrefix`, why did you add this one here?

Reply:
my bad,removed.